src/main/java/org/zalando/nakadi/domain/EventTypeOptions.java

```diff
@@ -1,10 +1,7 @@
 package org.zalando.nakadi.domain;
 
-import javax.annotation.Nullable;
-
 public class EventTypeOptions {
 
-    @Nullable
     private Long retentionTime;
```

Comment:
In that case it will be hard-code but I think it is better to have possibility to change it.